libs/common-system/src/test/java/org/codice/ddf/configuration/AbsolutePathResolverTest.java

```diff
@@ -17,30 +17,43 @@
 import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+import com.google.common.collect.ImmutableList;
 import java.io.File;
 import java.nio.file.Paths;
+import java.util.List;
 import org.junit.Test;
 
 public class AbsolutePathResolverTest {
 
+  private static final List<String> SYSTEM_PROPERTIES = ImmutableList.of("ddf.home", "karaf.home");;
```

Comment:
❗️ This is premature optimization. We don't need all this support. I can't think of a case where we would need more than `ddf.home` and `karaf.home`.